source/features/pr-easy-toggle-files.tsx

```diff
@@ -3,13 +3,25 @@ import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 
-function toggleFile(event: Event): void {
+const toggleFileButtonSelector = '[aria-label="Toggle diff contents"]';
+
+function toggleFile(event: MouseEvent): void {
 	const elementClicked = event.target as HTMLElement;
-	const headerBar = elementClicked.closest<HTMLElement>('.file-header.file-header--expandable')!;
+	const headerBar = elementClicked.closest<HTMLElement>('.file-header')!;
 
 	// The clicked element is either the bar itself or one of its 2 children
 	if (elementClicked === headerBar || elementClicked.parentElement === headerBar) {
-		select('[aria-label="Toggle diff contents"]', headerBar)!.click();
+		if (event.altKey) {
```

Comment:
Should we also check for `event.isTrusted` here as a precaution?

Reply:
Precaution against what?

Reply:
Clicks generated by a script and not the user, e.g. in another RGH feature, another GH-related extension, a userscript etc.

Reply:
That’s fine, there’s no meaning to “alt click” on buttons, so any fake mouse events with alt are likely to want our behavior. 

Generally I think the only reason to use isTrusted for us is to avoid clicks we ourselves generate (like in toggle-all-with-alt)